lib/ai/chat.go

```diff
@@ -170,7 +171,8 @@ top:
 
 		// if we can parse it, return the parsed payload, otherwise return a non-streaming message
 		var c CompletionCommand
-		err = json.Unmarshal([]byte(payload), &c)
+		fmt.Printf("Command payload: %s", payload)
```

Comment:
should this be a printf? maybe a debug log?

Reply:
my bad, debugging leftover

```suggestion
```